homeassistant/components/camera/mjpeg.py

```diff
@@ -44,6 +44,14 @@
 async def async_setup_platform(hass, config, async_add_entities,
                                discovery_info=None):
     """Set up a MJPEG IP Camera."""
+    # Filter header errors from urllib3 due to a urllib3 bug
+    urllib3_logger = logging.getLogger("urllib3.connectionpool")
```

Comment:
what's with all the red markers?

Reply:
I was wondering that myself. They're just normal parentheses 🤷‍♂️